portfolio/src/main/webapp/lucygallery.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Lucy's Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body bgcolor = "#66819c">
```

Comment:
We generally don't put spaces around the equals sign in HTML. It's unlikely to break things in modern browsers, but still a good style guideline for readability.

```suggestion
  <body bgcolor="#66819c">
```